### PR TITLE
NAS Envelopenet Suggestion and Job Example

### PR DESCRIPTION
**Pull request by garganubhav**
This PR is about NAS algorithm Envelopenet https://arxiv.org/pdf/1803.06744.pdf. I will add Training container PR separately. TODO: Multiple studyjobs and trials support.

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/katib/425)

### CODE REVIEW

**Comment by garganubhav** [comment]
@richardsliu @YujiOshima Plz review

**Comment by garganubhav** [comment]
@richardsliu @andreyvelich Please review and close this.

File: cmd/suggestion/nasenvelopenet/main.py [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+from pkg.suggestion.nasenvelopenet_service import EnvelopenetService
+from pkg.suggestion.types import DEFAULT_PORT
+from logging import getLogger, StreamHandler, INFO, DEBUG
+
+
+_ONE_DAY_IN_SECONDS = 60 * 60 * 24
+
+
+def serve():
+    print("NAS Envelopenet Suggestion Service")
```
Comment (andreyvelich) [inline cmd/suggestion/nasenvelopenet/main.py]: I think, you should change print to logging, like here: https://github.com/kubeflow/katib/blob/master/pkg/suggestion/v1alpha1/nasrl_service.py#L51

**Comment by richardsliu** [comment]
Can you include a README with this example? Also is it based on this paper? https://arxiv.org/pdf/1803.06744.pdf If so please link it here.

**Comment by garganubhav** [comment]
@richardsliu done

**Comment by garganubhav** [comment]
@richardsliu Plz merge

**Pull request merged**